security_drone_user_app/lib/presentation/pages/view_thief_page.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:security_drone_user_app/logic/thief_page_bloc.dart';
+import 'package:security_drone_user_app/presentation/text_section.dart';
+
+import '../image_banner.dart';
+
+class ViewThiefPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return ViewThiefPageState();
+  }
+}
+
+class ViewThiefPageState extends State<ViewThiefPage>{
+
+  @override
+  Widget build(BuildContext context){
+    // ignore: close_sinks
+    ThiefPageBloc _bloc = ThiefPageBloc();
+
+    return Scaffold(
+      body: ThiefListClass(_bloc),
+      floatingActionButton: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          FloatingActionButton(
+            child: Icon(Icons.refresh),
+            onPressed: () => _bloc.add(
+              RefreshThiefEntries(),
+            ),
+          ),
+        ],
+      )
+    );
+  }
+}
+
+// ignore: must_be_immutable
+class ThiefListClass extends StatelessWidget {
+  ThiefPageBloc _bloc;
+
+  ThiefListClass(this._bloc);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Security Drone"),
+        actions: [
+          ImageBanner('assets/images/icon.jpg')
+        ],
+      ),
+      body: ThiefList(_bloc),
+    );
+  }
+}
+
+// ignore: must_be_immutable
+class ThiefList extends StatelessWidget {
+  ThiefPageBloc _bloc;
+
+  ThiefList(this._bloc);
+
+  @override
+  Widget build(BuildContext context){
+    return SingleChildScrollView(
+      padding: EdgeInsets.all(10.0),
+      child: BlocConsumer<ThiefPageBloc, ThiefPageState>(
+        cubit: _bloc,
+        buildWhen: (prev, curr){
+          return curr.entries.isNotEmpty;
+        },
+        // ignore: missing_return,
+        builder: (context, state) {
+          if (state is ShowThiefEntries) {
+            var builder = ListView.builder(
+                physics: ScrollPhysics(),
+                shrinkWrap: true,
+                padding: EdgeInsets.all(10.0),
+                itemCount: state.entries.length,
+                itemBuilder: (context, index) {
+                  return Card(
+                    child: ListTile(
+                      contentPadding: EdgeInsets.symmetric(horizontal: 50),
+                      title: state.entries[index].image,
+                      subtitle: Text(index.toString()),
+                      onTap: () => _bloc.add(ThiefEntryClicked(index)),
+                    ),
+                  );
+                }
+            );
+            return builder;
+          }
+          else if (state is ShowThiefEntry){
+            var index = state.focusedIndex;
+            var display = Column(
+              children: [
+                state.entries[index].image,
+                TextSection(
+                  state.entries[index].date.toString(),
```

Comment:
Don't use toString on date, use formatting that you'd like to show.
Reference: https://stackoverflow.com/questions/16126579/how-do-i-format-a-date-with-dart